django_tasks/backends/database/models.py

```diff
@@ -63,7 +63,7 @@ class DBTaskResult(GenericBase[P, T], models.Model):
         max_length=max(len(value) for value in ResultStatus.values),
     )
 
-    enqueued_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Question: What's the reason for this change?

Reply:
`enqueued_at` was getting set at insert rather than at model creation time. We would have to do an insert and an update to set the signature. Now whether or not enqueued_at should represented the time `.enqueue` was called or the time the record was inserted, I'm not sure about.